### ISSUE TITLE
Missing esm module file

### ISSUE THREAD

**Issue by More4ever** [ghybs/leaflet-defaulticon-compatibility#5]
Hello

Hope you are well

I migrate Laravel application from Mix.js to Vite.js

When I try to import package I receive such error 

```
[commonjs--resolver] Failed to resolve entry for package "leaflet-defaulticon-compatibility". 
The package may have incorrect main/module/exports specified in its package.json.
```

values from curren published npm package build

```
  "main": "dist/leaflet-defaulticon-compatibility.js",
  "module": "src/Icon.Default.compatibility.js",
```
Current file structure

![Screenshot 2023-07-07 at 10 52 34](https://github.com/ghybs/leaflet-defaulticon-compatibility/assets/1040629/5b189c61-8db2-418c-a4b4-def80a857ec4)

As you see `src` dirrectory is not present

**Comment by srosset81** [comment]
I have the same error with a React app bundled with Vite.js

**Comment by ghybs** [comment]
Hi,

I published version 0.1.2 that should make that file available now: https://unpkg.com/browse/leaflet-defaulticon-compatibility@0.1.2/src/Icon.Default.compatibility.js

Please let me know if it works with your setup?

**Comment by srosset81** [comment]
It works great, thank you @ghybs !!
